Sanguine Soul/sanguinesoul.asl

```diff
@@ -13,39 +13,42 @@ startup
 {
 	Assembly.Load(File.ReadAllBytes("Components/asl-help")).CreateInstance("Basic");
 	vars.Helper.GameName = "Sanguine Soul";
+	vars.Helper.Settings.CreateFromXml("Components/SanguineSoul.Settings.xml");
+
+	vars.CompletedSplits = new HashSet<string>();
+
 	vars.Helper.AlertRealTime();
-    vars.Helper.Settings.CreateFromXml("Components/SanguineSoul.Settings.xml");
 }
 
 init
 {
     // `fname` is the index into the GNames array
-    vars.CachedFNames = new Dictionary<long, string>();
-    vars.ReadFName = (Func<long, string>)(fname => 
+    var cachedFNames = new Dictionary<int, string>();
```

Comment:
Why change the casing?

Reply:
It's changed to a local (that is captured by the function). Since the cache is not accessed anywhere else in the code, it does not need to be stored in `vars`.